### PR TITLE
[Doc] Removed reference to texscreen in BackBufferCopy docs

### PR DESCRIPTION
**Pull request by clayjohn**
A discord user noticed the reference to texscreen.

### CODE REVIEW

File: doc/classes/BackBufferCopy.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="BackBufferCopy" inherits="Node2D" category="Core" version="3.1">
 	<brief_description>
-		Copies a region of the screen (or the whole screen) to a buffer so it can be accessed with the texscreen() shader instruction.
+		Copies a region of the screen (or the whole screen) to a buffer so it can be accessed with ``SCREEN_TEXTURE`` in the ``texture()`` function.
```
Comment (akien-mga) [inline doc/classes/BackBufferCopy.xml]: You should use `[code]texture()[/code]`, not <code>``</code>.

**Comment by clayjohn** [comment]
Oh whoops. Im too used to the tutorial formatting I guess. Ill fix this up tonight.

**Comment by clayjohn** [comment]
Should be good to go now @akien-mga Thanks again for reviewing!!

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.0.7.

**Pull request merged**